kafka-ui-contract/src/main/resources/swagger/kafka-ui-api.yaml

```diff
@@ -1878,3 +1910,21 @@ components:
         - connect
         - status
 
+    PartitionsIncrease:
+      type: object
+      properties:
+        totalPartitionsCount:
+          type: integer
+      required:
+        - totalPartitionsCount
+
```

Comment:
Shouldn't we validate `totalPartitionsCount` value? For example, is it > 0?

Reply:
And please check that they are bigger than current